tests/framework/microvm.py

```diff
@@ -345,6 +347,7 @@ def init_snapshot_api(self):
             self._api_session
         )
 
+    # pylint: disable=too-many-statements
```

Comment:
I'm so sorry you were unlucky enough to hit the limit with just **one** line added... but you were nonetheless.

Please split the function instead of disabling the lint.